core/search_index.py

```
@@ -340,16 +344,28 @@ def _cancelled() -> bool:
 
                 if all_cf_prefixes:
                     _ph = ','.join('?' * len(all_cf_prefixes))
-                    conn.execute(f"DELETE FROM jobs WHERE prefix NOT IN ({_ph})", tuple(all_cf_prefixes))
-                    conn.execute(f"DELETE FROM indexed_dirs WHERE kind='cf' AND prefix NOT IN ({_ph})", tuple(all_cf_prefixes))
+                    conn.execute(  # nosec B608
+                        f"DELETE FROM jobs WHERE prefix NOT IN ({_ph})",
```

Ruff (0.15.12)

[error] Possible SQL injection vector through string-based query construction

(S608)